### ISSUE TITLE
Extension Error: Failed to construct 'URL': Invalid URL

### ISSUE THREAD

**Issue by enjikaka** [GoogleChrome/lighthouse#1208]
**Lighthouse Version**: 1.3.0
**Chrome Version**: 57.0.2952.0
**Error Message**: Failed to construct 'URL': Invalid URL
**Stack Trace**:
 ```TypeError: Failed to construct 'URL': Invalid URL
    at artifacts.DateNowUse.usage.filter.err (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:1046:39)
    at Array.filter (native)
    at Function.audit (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:1045:48)
    at Promise.resolve.then._ (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:11365:20)```

**Comment by wardpeet** [comment]
@enjikaka could you provide us with an url please.
do you have any chrome plugins installed? Do you have any options deselected on the extension?

// dupe of #1194

**Comment by enjikaka** [comment]
@wardpeet I tried with several URLs now and all failed. I disabled all extensions but Lighthouse and then it worked. I then enabled one extension at a time and did new runs. It failed when I finally enabled Redux DevTools. I also tried only having Redux and Lighthouse enabled, and then it fails with this error.

Lighthouse Version: 1.3.0
Chrome Version: 57.0.2952.0
Redux DevTools Version: 2.11.1.1

( https://github.com/zalmoxisus/redux-devtools-extension )
